holoviews/tests/ui/bokeh/test_hover.py

```diff
@@ -222,13 +211,51 @@ def test_hover_tooltips_dimension_unit(serve_hv, hover_tooltip):
 
     page = serve_hv(hv_curve)
     hv_plot = page.locator(".bk-events")
-    wait_until(lambda: expect(hv_plot).to_have_count(1), page=page)
+    expect(hv_plot).to_have_count(1)
     bbox = hv_plot.bounding_box()
 
     # Hover over the plot
     page.mouse.move(bbox["x"] + bbox["width"] / 2, bbox["y"] + bbox["height"] / 2)
     page.mouse.up()
 
-    wait_until(lambda: expect(page.locator(".bk-Tooltip")).to_have_count(1), page=page)
+    expect(page.locator(".bk-Tooltip")).to_have_count(1)
 
     expect(page.locator(".bk-Tooltip")).to_contain_text("Amplitude (µV): 10")
+
+
+@pytest.mark.usefixtures("bokeh_backend")
+def test_hover_tooltips_rasterize_server_hover(serve_hv, rng):
+    import datashader as ds
+
+    from holoviews.operation.datashader import rasterize
+
+    df = pd.DataFrame({
+        "x": rng.normal(45, 1, 100),
+        "y": rng.normal(85, 1, 100),
+        "s": 1,
+        "val": 10,
+        "cat": "cat1",
+    })
+    img = rasterize(hv.Points(df), selector=ds.first("val")).opts(tools=["hover"])
+
+    page = serve_hv(img)
+    hv_plot = page.locator(".bk-events")
+    expect(hv_plot).to_have_count(1)
+    bbox = hv_plot.bounding_box()
+
+    # Hover over the plot, first time the hovertool only have null
```

Comment:
We could make it so it does not return null, but a random point.

Reply:
Current behavior seems correct to me.